kodilla-rps/src/main/java/Round.java

```diff
@@ -0,0 +1,46 @@
+public class Round {
+
+    ActionType playerAction;
+    ActionType computerAction;
+    private Player player;
+    private Computer computer;
+
+    public Round(Player player, Computer computer) {
+        this.player = player;
+        this.computer = computer;
+        this.playerAction = player.playerAction();
+        this.computerAction = computer.computerAction();
+    }
+
+    public Result checkingWinnerOfRound() {
+        if (playerAction == computerAction)
+            return Result.TIE;
+        switch (playerAction) {
+            case ROCK:
+                return (computerAction == ActionType.PAPER ? Result.LOSE : Result.WIN);
+            case PAPER:
+                return (computerAction == ActionType.SCISSORS ? Result.LOSE : Result.WIN);
+            case SCISSORS:
+                return (computerAction == ActionType.ROCK ? Result.LOSE : Result.WIN);
+        }
+        return Result.LOSE;
+    }
+
+    public void result() {
```

Comment:
this class has 2 functionalities:
- it checks who wins round
- it display messages.

I think it is too much for one simple class

Reply:
done

Reply:
done